dev/doc/release-process.md

```diff
@@ -1,169 +1,159 @@
-# Release process #
-
-## As soon as the previous version branched off master ##
-
-In principle, these steps should be undertaken by the RM of the next
-release. Unfortunately, we have not yet been able to nominate RMs
-early enough in the process for this person to be known at that point
-in time.
-
-- [ ] Create a new issue to track the release process where you can copy-paste
-  the present checklist from `dev/doc/release-process.md`.
-- [ ] Change the version name to the next major version and the magic
-  numbers (see [#7008](https://github.com/coq/coq/pull/7008/files)).
+# Release checklist #
+
+## When the release managers for version `X.X` get nominated ##
```

Comment:
```suggestion
## When the release managers for version `X.X` gets nominated ##
```

Reply:
Notice the "s" at "release managers" which explains the no "s" at "get".

Reply:
oops